Modules/ITS/src/ITSTrackTask.cxx

```diff
@@ -628,6 +723,21 @@ void ITSTrackTask::createAllHistos()
   addObject(hTrackPtVsPhi);
   formatAxes(hTrackPtVsPhi, "#it{p}_{T} (GeV/#it{c})", "#phi", 1, 1.10);
   hTrackPtVsPhi->SetStats(0);
+
+  for (int l = 0; l < NLayer; l++) {
+    //sensor
+    hdxySensor[l] = std::make_unique<TH2D>(Form("hdxy%dSensor",l), Form("ChipID vs dxy, Layer %d",l), 
+                                           1000, -0.05, 0.05, ChipBoundary[l+1] - ChipBoundary[l], ChipBoundary[l], ChipBoundary[l+1]);
```

Comment:
do you really need 1000 bins on the X axis? Same comment for `hdz` histo. Bin width is 1 um like this, maybe 5 um are enough (i.e. comparable with the ALPIDE spatial reso).

Reply:
Make the y axis bin with the chipID in the center: 
`ChipBoudary[l+1] - ChipBoundary[l] + 1, ChipBoundary[l] - 0.5, ChipBoundary[l+1] - 0.5`

Note the -0.5 also for the upper limit because for example chip 108 belongs already to L1 (and not to L0) if we start counting chipID from 0. Maybe make again the calculation but it should be correct.